tests/base/test_monitors.py

```diff
@@ -30,6 +30,12 @@ def test_keyring_exists(self, mon_containers, client):
         if client.exec_inspect(command)['ExitCode'] != 0:
             raise AssertionError(result)
 
+    def test_monmap_exists(self, mon_containers, client):
+        command = client.exec_create(mon_containers, cmd='stat /etc/ceph/monmap-ceph')
```

Comment:
Correct. Now I see where the `ceph.monmap` comes from. It's only on the demo images. I'm going to normalize that.